test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -110,7 +111,9 @@ declare module 'test-case-reporting' {
       duration_ms: number;
     }
 
-    export interface BigJoin extends Build, Job, TestCase, TestRun {}
+    export interface BigJoin extends Build, Job, TestCase, TestRun {
```

Comment:
I know this wasn't done in this PR but I'm not a big fan of this name??? 😅
What is it used for exactly? Give it a descriptive name

Reply:
Lmao I missed this name. `TestRunWithJob` or something maybe

Reply:
`JoinOfAllTables`? Because it's actually a `TestRunWithJobAndBuildAndTestCase`...

Reply:
The type doesn't need to explicitly mention it's based on DB tables or a join. TestRunWithMeta, TestRunWithJobAndBuild would both be more appropriate imo

Reply:
Well, we only need the type because we're using a DB table join -- if we weren't doing that, we wouldn't need this type -- so I don't think that coupling would be too bad.

I think I like `TestRunWithJobAndBuild` or `TestRunWithFullInfo`. Thoughts?

Reply:
(We can do that in a separate PR, pushing for now...)

Reply:
`TestRunWithJobAndBuild` SGTM

Reply:
Done, PTAL at https://github.com/ampproject/amp-github-apps/pull/973